src/k8s-engine/images/kimImageProcessor.ts

```diff
@@ -0,0 +1,256 @@
+import { Buffer } from 'buffer';
```

Comment:
Why do we have this full kim implementation? The abstraction layer will come into handy when we add Docker CLI support and run Moby instead of containerd. I'm just curious why we have this Kim implementation if we aren't planning on using it.

Reply:
Good point - a good final step for this PR is to remove the kim image-processor. As discussed on 2021-10-01 we can still ship the binary. There might be people who've written tools that depend on it, but don't need the internal class.